### ISSUE TITLE
update all VM images in cloud stores to ubuntu 24 as soon as 24.04.1 is out

### ISSUE THREAD

**Issue by szaimen** [nextcloud/all-in-one#4775]
Should be `August 15, 2024.`

- [x] test if database dump works correctly

https://www.cyberciti.biz/faq/how-to-upgrade-from-ubuntu-22-04-lts-to-ubuntu-24-04-lts/

TODO:
- [ ] aws https://github.com/nextcloud-releases/all-in-one/pull/52
- [ ] digitalocean https://github.com/nextcloud-releases/all-in-one/pull/53
- [x] ~~google-cloud~~ https://github.com/nextcloud-releases/all-in-one/commit/9895a61fbd3416489f1f73b15b70a8391411c719
- [x] ~~linode~~ https://github.com/nextcloud-releases/all-in-one/commit/fb0fe3fffff01d1e118c286d416a2ecc082f80f0
- [ ] microsoft-azure https://github.com/nextcloud-releases/all-in-one/pull/54
    - [ ] https://github.com/nextcloud/all-in-one/issues/6621
- [x] ova https://github.com/nextcloud-gmbh/sysadmin/issues/1586
    - [ ] bug: sshd fails to start: https://www.garron.me/en/linux/sshd-no-hostkeys-available-exiting.html

**Comment by thibaultmol** [comment]
I don't know if it's related, but I noticed that the vm image linked on this page is like 2 years old right now: https://nextcloud.com/install/#instructions-server

**Comment by szaimen** [comment]
Yes, it is still using Ubuntu 22.04 LTS as base. It will also be updated then.

**Comment by szaimen** [comment]
The ova image was just updated

**Comment by snakwu** [comment]
> ova 映像刚刚更新
升级到Ubuntu 24.04.1 LTS后，无法使用了
![image](https://github.com/user-attachments/assets/12aca6b2-b292-4c67-ba96-d84c9d0b5be7)

**Comment by snakwu** [comment]
![image](https://github.com/user-attachments/assets/803f1b62-5d73-49be-b5c4-4d71687152f6)

**Comment by szaimen** [comment]
Will continue in https://github.com/nextcloud/all-in-one/issues/7883
